java/daikon/chicory/ChicoryPremain.java

```diff
@@ -44,6 +44,18 @@
    **/
   private static boolean doPurity = false;
 
+  // This @SuppressWarnings is for the two lines
+  // Runtime.decl_writer = new DeclWriter (Runtime.dtrace);
+  // Runtime.dtrace_writer = new DTraceWriter (Runtime.dtrace);
+  //
+  // Runtime.dtrace is @GuardedBy("itself") because in the Runtime class,
+  // dtrace.close() is called only when the monitor of dtrace is held
```

Comment:
I clarified the comment - I meant the entire operation of closing a trace, which involves calling the println() and close() methods on it.